Find a functionally equivalent code:android.content.res.TypedArray.getIndex:COMMENT
Method Modifier: public      
Comment:/**
 * Return an index in the array that has data.
 *
 * @param at The index you would like to returned, ranging from 0 to
 * {@link #getIndexCount()}.
 *
 * @return The index at the given offset, which can be used with
 * {@link #getValue} and related APIs.
 */

@@ -1,3 +1,6 @@
 {
+    if (mRecycled) {
+        throw new RuntimeException("Cannot make calls to a recycled instance!");
+    }
     return mIndices[1 + at];
 }

